docs/CustomUIA.md

```diff
@@ -0,0 +1,115 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT Licence. -->
+
+## Custom UI Automation properties
+Axe Windows supports the inspection of [custom UI Automation properties](https://docs.microsoft.com/en-gb/windows/win32/winauto/uiauto-custompropertieseventscontrolpatterns). The custom properties for which Axe Windows should register are described in a user-supplied text file that is applied to all inspected UI Automation elements.
+
+### Configuration file format
+Custom UI Automation configuration data is specified in a [JSON](https://en.wikipedia.org/wiki/JSON#Syntax) formatted text file. The file consists of one object with the following possible attributes:
+
+#### metadata
+This attribute, if specified, can contain values to describe the file as determined by the user. It may be useful, for instance, to store version or authorship information. Example:
+
+```
+  "metadata": {
+    "version": "1.0",
+    "modified": "1970-01-01"
+  }
+```
+
+#### properties
+This attribute contains an array of objects, one per defined custom UI Automation property. Each property object contains the following fields:
+
+Attribute | Description
+--- | ---
+guid|The globally unique identifier of this property.
+programmaticName|A textual description of this property.
+uiaType|The data type of this property's value, one of `string`, `int`, `bool`, `point`, `element`, or `enum`.
```

Comment:
Is `enum` accurate? The [UI Automation docs for custom properties](https://docs.microsoft.com/en-us/windows/win32/winauto/uiauto-designingcustompropseventpatterns#designing-custom-properties) suggests that there should be a `double` type and should not be an `enum` type.

Reply:
Ah, sorry, hadn't read far enough down to see that it's using an int representation. But is it intentional for `double` to be omitted?

Reply:
Not intentional!